### PR TITLE
Issue#42

### PR DESCRIPTION
**Pull request by prakharjoshi**
This includes the content type layer for the testing environment for local testing and also there are few new functional tests to confirm the functionality of add-on :)

### CODE REVIEW

**Comment by prakharjoshi** [comment]
@lentinj @gforcada  Thanks it worked !! Now we can merge it I guess

File: README.rst [hunk 1]
```diff
@@ -1,3 +1,25 @@
...
+Why this add-on is required ?
+====================
+
+This add-on is required for two main reasons :-
+1) It removes the CMFDefault dependencies from the Plone (From HTML Filtering)
+2) It is quite faster transform then the earlier one as it uses lxml for filtering HTML
+
+====================
+What is does ?
+====================
```
Comment (gforcada) [inline README.rst]: Headers should always adjust to the length of the titles, so

```
==========
Short name
==========

and

=====================
Super duper long name
=====================
```

File: src/experimental/safe_html_transform/tests/test_filter.py [hunk 2]
```diff
@@ -24,36 +24,41 @@ def setUp(self):
...
             self.browser.url.endswith('plone_control_panel'))
 
     def test_status_msg_after_cancel(self):
-        self.browser.open(self.portal_url + '/@@safe_html_transform-settings')
+        self.browser.open(self.portal_url + '/@@filter-controlpanel')
         self.browser.getControl(name="form.buttons.cancel").click()
         self.assertTrue(
             'Changes canceled.' in self.browser.contents)
 
+    def test_login(self):
```
Comment (gforcada) [inline src/experimental/safe_html_transform/tests/test_filter.py]: This test does not belong here, experimental.safe_html_transform has nothing to do with logins, right? :-)

**Comment by gforcada** [comment]
I added some comments, to make the pull request extra nice.

Some `git rebase --interactive` to merge commits would be extra nice as well, so the git history looks cleaner and easier to follow

**Pull request merged**